scio-core/src/main/scala/com/spotify/scio/coders/instances/GuavaCoders.scala

```diff
@@ -0,0 +1,21 @@
+package com.spotify.scio.coders.instances
+
+import java.io.{InputStream, OutputStream}
+
+import com.google.common.hash.BloomFilter
+import com.spotify.scio.coders.Coder
+import com.google.common.{hash => g}
+import org.apache.beam.sdk.coders.AtomicCoder
+
+class GuavaBloomFilterCoder[T](implicit val funnel: g.Funnel[T])
```

Comment:
https://github.com/spotify/scio/blob/master/scio-core/src/main/scala/com/spotify/scio/hash/ApproxFilter.scala#L337

We have this already but private, maybe move it here?

Reply:
Realized I could `xmap` to the ApproxFilter wrapper, so updated.